salt/modules/drac.py

```diff
@@ -174,16 +174,16 @@ def list_users():
     users = {}
     _username = ''
 
-    for i in range(1, 12):
+    for idx in range(1, 17):
```

Comment:
42!
Could you please move this magic number into a constant, so everyone would know what it means?